polkadot-parachains/integritee-runtime/src/lib.rs

```diff
@@ -798,6 +755,7 @@ mod benches {
 		[pallet_collective, TechnicalCommittee]
 		[pallet_democracy, Democracy]
 		[pallet_multisig, Multisig]
+		[pallet_preimage, Preimage]
```

Comment:
New way of submitting preimages of proposals as far as I understand. I have not checked how to use it yet.